modules/account-lib/test/unit/coin/algo/transactionBuilder/assetTransferBuilder.ts

```diff
@@ -208,16 +196,12 @@ describe('Algo Asset Transfer Transaction Builder', () => {
       txInfo.genesisHash.should.equal('SGO1GKSzyE7IEPItTxCByw9x8FmnrCDexi9/cOUJOiI=');
     });
 
-    it('should build a valid disable token transaction and re-calcualte minimum fee', async function() {
+    it('should build a valid disable token transaction and re-calcualte minimum fee', async function () {
       const firstRound = 167;
       const lastRound = 1167;
       const fee = 1000;
       const tokenId = 1;
       const closeRemainderTo = 'SP745JJR4KPRQEXJZHVIEN736LYTL2T2DFMG3OIIFJBV66K73PHNMDCZVM';
-      const {
-        networks: { testnet },
-      } = AlgoResources;
-      const { genesisHash, genesisID } = testnet;
```

Comment:
Same here

Reply:
This lines weren't used and they be marked by lint as an error